### PR TITLE
Some intial simplified components + stories

### PR DESCRIPTION
**Pull request by infiniteluke**
- Adding a couple basic components just to get started.
- Added [Storybook](https://github.com/kadirahq/react-storybook) so that @tsmith512 can start working on styles without the rest of the components built. To test this:
  - Pull this branch
  - `npm i`
  - `npm run storybook`
  - Edit the color in `src/components/Card/Card.scss`
  - See update

### CODE REVIEW

**Comment by infiniteluke** [comment]
The codecov commenting will be enabled by this PR.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+engines:
+  eslint:
+    enabled: false
+
+ratings:
+  paths:
+  - "**.js"
```
Comment (flipactual) [inline .codeclimate.yml]: Can this be indented another level? That's how I tend to write yml, but tbh I have no idea which is "right"

File: .storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { configure } from '@kadira/storybook';
+
+function loadStories() {
+  require('../src/components/stories');
+}
+
+configure(loadStories, module);
```
Comment (flipactual) [inline .storybook/config.js]: Can `loadStories` be made anonymous/arrow and just passed in to `configure`?

File: src/components/Card/Card.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.card.selected {
```
Comment (flipactual) [inline src/components/Card/Card.scss:1]: Maybe `.card.is-selected`?

File: src/components/Card/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Card.scss';
+
+const Card = ({ value, selected }) => (
+  <div className={`card ${selected ? 'selected' : ''}`}>
```
Comment (flipactual) [inline src/components/Card/index.js]: Might be good to go ahead and pull in [`classnames`](https://github.com/JedWatson/classnames)

File: src/components/Card/stories.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import Card from './';
+
+const value = 13;
```
Comment (flipactual) [inline src/components/Card/stories.js]: I still prefer to caps lock my `const`s, but I'm just chiming in, not concerned.

File: src/index.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<html manifest="feelings.appcache">
```
Comment (flipactual) [inline src/index.html:1]: I'm dead

File: test/Card.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import test from 'ava';
+import React from 'react';
+import Card from '../src/components/Card';
+import { render } from 'enzyme';
```
Comment (flipactual) [inline test/Card.js:4]: neat

**Review by flipactual — CHANGES_REQUESTED** [review]
Few nits to discuss for maximum funness.

File: src/components/Card/index.js (hunk 4 above)
Comment (infiniteluke) [reply src/components/Card/index.js]: I wondered. Good call.

File: src/components/Card/Card.scss (hunk 3 above)
Comment (infiniteluke) [reply src/components/Card/Card.scss:1]: I agree this sounds better, but I'm trying to keep classnames/markup as backwards compatible as possible to make @tsmith512's life easy.

File: src/index.html (hunk 6 above)
Comment (infiniteluke) [reply src/index.html:1]: No more feelings

**Comment by infiniteluke** [comment]
@flipactual Ready for review again.

File: src/components/Card/Card.scss (hunk 3 above)
Comment (flipactual) [reply src/components/Card/Card.scss:1]: 👌

**Review by flipactual — APPROVED** [review]

File: src/index.html (hunk 6 above)
Comment (fluxsauce) [reply src/index.html:1]: ☠️

**Pull request merged**